inc/php_basic_auth/namespace.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Altis\Security\PHP_Basic_Auth;
+
+use function Altis\get_config;
+use function Altis\get_environment_type;
+
+function bootstrap() {
+	add_action( 'hmauth_action_before_dev_env_check', __NAMESPACE__ . '\\check_environment_credentials' );
+}
+
+/**
+ * Set the HM_BASIC_AUTH_USER and HM_BASIC_AUTH_PW constants based on the
+ * parameters passed in the composer.json file.
+ */
+function define_credentials() {
+	// Get the config values.
+	$config = get_config()['modules']['security']['php-basic-auth'];
```

Comment:
Just realised this will get run if `get_config()['modules']['security']['php-basic-auth']` is `true` - so you'd always get the warning even if you'd defined `HM_BASIC_AUTH_USER` etc.

I think it'd be enough to add an `is_array()` check here:

```suggestion
	$config = get_config()['modules']['security']['php-basic-auth'];
	
	if ( ! is_array( $config ) ) {
		return;
	}
```

Reply:
Sorry, I totally wasn't clear - I meant in addition to the checks you already had. I'll add them back in and be more specific next time. My bad.